qa/suites/fs/upgrade/featureful_client/upgraded_client/from_nautilus/tasks/1-client.yaml

```diff
@@ -0,0 +1,5 @@
+nuke-on-error: false
+overrides:
+  nuke-on-error: false
+tasks:
+- print: "**** done nautilus client"
```

Comment:
we can remove this yaml, no?

Reply:
I think it's okay to remove it. Just copied it from other place.